qiling/os/memory.py

```diff
@@ -37,10 +40,15 @@ def __init__(self, ql: Qiling):
 
         max_addr = bit_stuff[ql.archbit]
 
-        #self.read_ptr = read_ptr
         self.max_addr = max_addr
         self.max_mem_addr = max_addr
 
+        # memory page size
+        self.pagesize = 0x1000
+
+        # make sure pagesize is a power of 2
+        assert self.pagesize & (self.pagesize - 1) == 0, 'pagesize has to be a power of 2'
```

Comment:
Maybe assert in @pagesize.setter is better choice?

Reply:
As a first step `pagesize` can be only configured statically. Other than that, the plan is to have `pagesize` set only at instance initialization (since allowing `pagesize` to change during runtime makes little sense to me), so I don't think there will be a setter for it.